script/draft-release/run.ts

```diff
@@ -89,7 +89,7 @@ function printInstructions(nextVersion: string, entries: Array<string>) {
 }
 
 /**
- * adds a number to the beginning fo each line and prints them in sequence
+ * adds a number to the beginning for each line and prints them in sequence
```

Comment:
```suggestion
 * adds a number to the beginning of each line and prints them in sequence
```

Reply:
> but I'm pretty sure it should be "of," not "for." Thanks!

I'm pretty sure you're 💯 right. How did I miss that? 😄

Reply:
I'm sorry, I was a bit too trigger happy here. If you'd like to open up another PR @olimart we'd happily review that as well.

Reply:
Done in #11065